Library/Homebrew/dev-cmd/bump-revision.rb

```diff
@@ -45,7 +45,14 @@ def bump_revision
         [checksum.hash_type, checksum.hexdigest]
       end
 
-      old = if hash_type
+      stable_block_given = formula.path.read.include? "  stable do"
+
+      old = if stable_block_given
+        # insert replacement revision before stable block
+        <<~EOS
+          stable do
+        EOS
```

Comment:
Does this work for `llvm` and insert before `head`?

Reply:
You're right, good catch. 
Probably I should've had a look at the order of the fields in the first place